pg_search/README.md

```diff
@@ -227,6 +227,7 @@ Finally, to enable the ICU tokenizer in development, pass `--features icu` to th
 First, start pgrx:
 
 ```bash
+# Note: If not using pg16, specify with (i.e. -- pg15, -- pg14, etc.)
```

Comment:
is it --pg15 or -- pg15 with a space?

Reply:
It is with a space.

I need to take a look because this line 

`cargo pgrx init --pg13=/opt/homebrew/opt/postgresql@13/bin/pg_config`

should be sufficient but now I need to pass the pg flag for everything, like running, testing and committing which is a pain

Reply:
Got it. Just wanted to make sure. No idea why you're facing this issue, normally `cargo pgrx init --pg13=...` should be enough.

If you want to make sure this does not happen, you could delete all Postgres versions from your system that aren't the ones you're working with. I've noticed specifically on macOS with Homebrew Postgres, sometimes there's issues.